src/app/components/forms/input/file-input/terra-file-input.component.spec.ts

```diff
@@ -99,8 +113,111 @@ describe('TerraFileInputComponent', () =>
         expect(component).toBeTruthy();
     });
 
-    afterAll(() =>
+    it('should accept TerraRegex.MIXED by default', () =>
+    {
+        expect(component.regex).toBe(TerraRegex.MIXED);
+    });
+
+    it('should have showPreview false by default', () =>
+    {
+        expect(component.inputShowPreview).toBeFalsy();
+    });
+
+    it('should have inputAllowFolders true by default', () =>
+    {
+        expect(component.inputAllowFolders).toBeTruthy();
+    });
+
+    it('should be an webImage if the extension is .jpg', () =>
+    {
+        expect(component.isWebImage(jpgFileName)).toBeTruthy();
+    });
+
+    it('should have publicUrl as value after selection', () =>
+    {
+        component.onObjectSelected(new TerraStorageObject(fileData.objects[0]));
+
+        expect(component.value).toBe(fileData.objects[0].publicUrl);
```

Comment:
```suggestion
        expect(component.value).toEqual(fileData.objects[0].publicUrl);
```
wäre das nicht genauer?